packages/fxa-auth-server/lib/senders/emails/partials/appBadges.ts

```diff
@@ -0,0 +1,21 @@
+export const appBadges = `<mj-section>
+    <mj-group>
+        <mj-column>
+            <mj-image css-class="app-badges" href="{{{iosUrl}}}" src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/5/ebdfb903-4e62-4aeb-9eae-26e1ef495049.png" alt="ios-badge"></mj-image>
+        </mj-column>
+        <mj-column>
+            <mj-image css-class="app-badges" href="{{{androidUrl}}}" src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/5/0dda16b4-21bb-4cc1-9042-2ebaaa7a9764.png" alt="android-badge"></mj-image>
+        </mj-column>
+    </mj-group>
+</mj-section>
+<mj-section>
+    <mj-column>
+        <mj-text css-class="secondary-text">
+            <% if(onDesktopOrTabletDevice){ %>
+                Or, install on <span href="http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device" color="#0a84ff" text-decoration="none" font-family="sans-serif">another version</span>
```

Comment:
We definitely don't want any URLs containing `localhost`. This should be some sort of variable passed down.